test/marketsDistribution.test.ts

```diff
@@ -21,3 +24,41 @@ describe("Markets distribution", () => {
     expect(marketDistribution.snapshotProposal).toBeTruthy();
   });
 });
+
+describe("Age 1 Epoch 2 approximation", () => {
+  const onChainDistribution = require("../distribution/fromDeprecatedScript/proofs-2.json");
+  const newDistribution = require("../distribution/proofs/proofs-2.json");
+
+  it("Should have the same number of users", () =>
+    expect(Object.keys(onChainDistribution.proofs).length).toEqual(Object.keys(newDistribution.proofs).length));
+
+  it("Should contains the same users", () => {
+    const newDistriUsers = Object.keys(newDistribution.proofs);
+    Object.keys(onChainDistribution.proofs).forEach((onChainUser) => expect(newDistriUsers).toContain(onChainUser));
+  });
+
+  it("Should derive of 7e-18 only", () => {
```

Comment:
From where does  the 7 wei value come from ?

Reply:
This is the biggest diff from prev script to the second one